source/user-manual/capabilities/vulnerability-detection/how-it-works.rst

```diff
@@ -39,18 +39,61 @@ The Vulnerability Detection module generates alerts when it detects new vulnerab
 
 .. _vuln_det_compatibility_matrix:
 
-Compatibility matrix
---------------------
-
-The following list shows the operating systems the CTI platform currently supports for vulnerability detection.
-
--  CentOS 5, 6, 7, 8, 9.
--  Red Hat 5, 6, 7, 8, 9.
--  Ubuntu 14 trusty, 16 xenial, 18 bionic, 20 focal, 22 jammy.
--  Debian 10 buster, 11 bullseye, 12 bookworm.
--  Amazon Linux 1, Amazon Linux 2, Amazon Linux 2023.
--  Arch Linux Rolling release.
--  SLES 11 server, SLED 11 desktop, SLES 12 server, SLED 12 desktop, SLES 15 server, SLED 15 desktop.
--  AlmaLinux 8, 9.
--  Windows XP and later.
--  macOS Sierra and later.
+
+Compatibility Matrix
+====================
+
+We are continuously adding new operating systems to our compatibility list. The following operating systems are currently supported for vulnerability detection, but this list is not exhaustive and will expand as we explore new updates.
```

Comment:
It seems like we don't support systems other than the ones expressed here.

I would re-phrase this to something along the lines of:

"""
The following list expresses operating systems that are officially supported by the Vulnerability Detection module. Other systems are supported, but full detection is not guaranteed.
"""

> [!NOTE]
> Maybe you can expand on what I've wrote

Reply:
Thanks!